src/schema-parser/openapi-schema-parser.ts

```diff
@@ -0,0 +1,117 @@
+import { SchemaParser, ParseSchemaInput, ValidateSchemaInput } from "../schema-parser";
+import type { AsyncAPISchema, SchemaValidateResult } from '../types';
+import Ajv, { ErrorObject, ValidateFunction } from "ajv";
+import * as fs from 'fs';
+import * as path from 'path';
+const toJsonSchema = require('@openapi-contrib/openapi-schema-to-json-schema');
+
+const schemaV3 = JSON.parse(fs.readFileSync(path.resolve(__dirname, './openapi/schema_v3.json'), 'utf8'));
```

Comment:
You should try to avoid using native nodeJS modules. Current OpenAPI and AVRO formats we can use in browser, RAML not (parser for that format uses java lib to convert raml to json-schema), so please pass content of `schema_v3.json` inside this file.

Reply:
Fixed!